proto/interchain_security/ccv/v1/shared_consumer.proto

```diff
@@ -62,11 +62,8 @@ message ConsumerParams {
     google.protobuf.Duration unbonding_period = 9
         [ (gogoproto.nullable) = false, (gogoproto.stdduration) = true ];
 
-    // The threshold for the percentage of validators at the bottom of the set who
-    // can opt out of running the consumer chain without being punished. For
-    // example, a value of 0.05 means that the validators in the bottom 5% of the
-    // set can opt out
-    string soft_opt_out_threshold = 10;
+    // !!! DEPRECATED !!! soft_opt_out_threshold is deprecated. see docs/docs/adrs/adr-015-partial-set-security.md
+    string soft_opt_out_threshold = 10 [deprecated = true];
```

Comment:
`ConsumerParams` is used to send data to the consumer chain "over the wire" - it is used in the consumer genesis.

Setting it to deprecated allows the field to exist but it will always be empty, thus maintaining backward compatibility.

We will not need to write a special parser to go from v4.2.0 `ConsumerParams` to some future version `ConsumerParams`.

Reply:
Agreed. This is why we did not use `reserved`.